.github/workflows/build.yml

```diff
@@ -2,6 +2,8 @@ name: Phing CI
 
 on:
   push:
+    branches:
+      - master
```

Comment:
The addition of branches/- master means they build will not run when someone pushes a branch to a forked repository. This means the build will only run when a Pull Request is submitted. I'd like to see the build run on my repo so I can correct any issues prior to submitting the Pull Request.